src/Recommendations/Process/Hydration.php

```diff
@@ -73,18 +74,33 @@ public function getPodcastEpisodeChapterById($id): PodcastEpisodeChapterModel
         return new PodcastEpisodeChapterModel($episode, $chapter);
     }
 
+    public function getExternalArticleByOriginalArticleId($id): ExternalArticle
+    {
+        list($originalArticleId, $relatedIndex) = explode('-', $id);
+        $relatedIndex = (int) $relatedIndex;
+
+/** @var ExternalArticle $episode */
+        // TODO: this uses sdk but it should really go through SingleItemRepository (doesn't have a method for this) or in any case through a cache
+        $externalArticle = $this->sdk->getRelatedArticles($originalArticleId)[$relatedIndex];
```

Comment:
Changing it back to this format:
```
mysql> SELECT * FROM Rules WHERE type='external-article';
+--------------------------------------+----------------------------------------------------+------------------+-----------+-------------+
| rule_id                              | id                                                 | type             | published | isSynthetic |
+--------------------------------------+----------------------------------------------------+------------------+-----------+-------------+
| 1a486488-f60c-410c-8545-8f0cac21b20d | 09560-https://doi.org/10.1126/scitranslmed.3001318 | external-article | NULL      |           1 |
+--------------------------------------+----------------------------------------------------+------------------+-----------+-------------+
```
preferring a readable id to an hash even if it's more work to parse.